public/scripts/slash-commands/SlashCommandClosure.js

```diff
@@ -232,9 +232,8 @@ export class SlashCommandClosure {
                     ;
                 }
 
-                let abortResult;
-                // eslint-disable-next-line no-cond-assign
-                if (abortResult = this.testAbortController()) {
+                let abortResult = this.testAbortController();
```

Comment:
It's really easy to mistype `a = b` vs `a == b`, and it will give you no visual indication on potentially unwanted reassignment